### DISCUSSION TITLE
Welcome to next-app Discussions!

### DISCUSSION THREAD

**Discussion by jgaa** [jgaa/next-app#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nmsleal** [comment]
Hi Jarle, I was pointed to this extremely interesting project by Claude Sonnet 4.5 when I prompted it about GTD workflows! I'm glad it did include it alongside several other options. As a GTD early adopter that tried so many tools to implement it, proudly started with a Palm OS PDA, I find this very promising! For now, I can only encourage you to keep up with it, though I'm worried this appears to be a one-member team. Will give it a try someday, when I get some time to become organized again... This is super cool stuff! Congrats!

**Reply by jgaa** [reply]
Hi, it's good if the AI's is picking it up. It's almost impossible today to create awareness around any project unless it's backed by big money and massive marketing.

I hope you get a chance to check it out and provide some feedback about what's good and what could be better ;)
